protocol/messenger_peersyncing_test.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 func TestMessengerPeersyncingSuite(t *testing.T) {
+	t.SkipNow() // FIXME
```

Comment:
Not important, but for non-suite tests we could use such approach for easier search in the future.
```suggestion
	t.SkipNow() // FIXME: s.T().Skip("flaky test")
```

Reply:
Yeah, in this case it is not flaky, as it fails all the time. I will introduce another type of comment in a followup: `s.T().Skip("failing test")`, wdyt?